lab-padget/lib/readfile.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+// Use node lib/readfile.js to test.
+const fs = require('fs');
+
+// push into array. console log that array.
+let myArray = []; // should have data 1, 2 and 3. Compare two arrays in test.
+// test the order is correct
+console.log(myArray, 'All the text!');
+// myArray.push(readFileOne.data1, readFileTwo.data2, readFileThree.data3);
+
+// File one: small
+
+exports.readThreeFiles = function(callback){
+  fs.readFile(`${__dirname}/../data/one.txt`, function(err, data) {
+    // console.log('1. Read file one: small');
+    // if(err) throw err;
+    // //console.log(data.toString());
+    // console.log(data.toString('ascii', 0, 8));
+    // console.log(data.toString('hex', 0, 8));
+    readFileOne(err, data);
+
+    // File two: medium
+    fs.readFile(`${__dirname}/../data/two.txt`, function(err, data) {
+      // console.log('2. Read file two: medium');
+      // if(err) throw err;
+      // //console.log(data.toString());
+      // console.log(data.toString('ascii', 0, 8));
+      // console.log(data.toString('hex', 0, 8));
+      readFileTwo(err, data);
+
+      // File three: large
+      fs.readFile(`${__dirname}/../data/three.txt`, function(err, data) {
```

Comment:
Though be careful when hard coding things in. When making these helper functions you want to be able to use this function for any files given.